src/Evolution/Systems/Cce/WorldtubeDataManager.hpp

```diff
@@ -20,6 +20,20 @@
 
 namespace Cce {
 
+namespace detail {
+
+template <typename InputTags>
+void puper(size_t& time_span_start, size_t& time_span_end,
```

Comment:
Rename to `set_non_pupped_members`. Also since this doesn't actually use the pupper, take all the reference arguments as `gsl::not_null`.